instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/DatabaseClientTracer.java

```diff
@@ -30,13 +29,17 @@
     extends BaseTracer {
   private static final String DB_QUERY = "DB Query";
 
-  protected final NetPeerAttributes netPeerAttributes;
+  protected final io.opentelemetry.instrumentation.api.tracer.net.NetPeerAttributes
```

Comment:
Why this change, @anuraaga ?

Reply:
Since `NetPeerAttributes` is deprecated and can't be used in import statement